lib/cadet/achievements/achievement.ex

```diff
@@ -0,0 +1,67 @@
+defmodule Cadet.Achievements.Achievement do
+  @moduledoc """
+  The Achievement entity stores metadata of a students' assessment
+  """
+  use Cadet, :model
+  use Arc.Ecto.Schema
+
+  alias Cadet.Achievements.{AchievementAbility, AchievementGoal}
+
+  schema "achievements" do
+    field(:inferencer_id, :integer)
```

Comment:
Yep! 

We are using a middleware to help us pre-process the achievements. 

As a result, this is technically our main identifier for the `Achievement` instead of our primary key ID

Reply:
Why not make this the primary key then, instead of an autoincrement column?

Reply:
Good Point!